persistence/src/main/scala/hmda/persistence/processing/SubmissionManager.scala

```diff
@@ -45,6 +46,7 @@ class SubmissionManager(submissionId: SubmissionId) extends HmdaActor {
   val supervisor = context.parent
   val hmdaFilingF = (supervisor ? FindHmdaFiling(period)).mapTo[ActorRef]
 
+  val submissionLarStats: ActorRef = context.actorOf(SubmissionLarStats.props(submissionId), s"submission-lar-stats-${submissionId.toString}")
```

Comment:
Oh, I might have figured out this one too. Is it to assign the name of the actor (`submission-lar-stats-${submissionId.toString}`) so that the `HmdaFileValidator` adn `HmdaFileParser` can find it?